src/styles/search-bar/search-bar.css

```diff
@@ -0,0 +1,46 @@
+.search-bar {
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  width: 100%;
+  margin: 20px 0;
+  border: 1px solid var(--border-gradient-color);
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.search-bar:hover {
+  box-shadow: var(--main-shadow);
+}
+
+.search-bar:focus-within {
+  background-image: linear-gradient(to top, var(--border-gradient-color), var(--bg-main-color));
+}
+
+.search-bar__icon {
+  background-image: url(/src/images/Search.svg);
+  position: absolute;
+  background-size: contain;
+  background-repeat: no-repeat;
+  border: none;
+  background-color: inherit;
+  pointer-events: none;
+}
+
+.search-bar__input {
+  border: none;
+  color: var(--text-color);
+  background-color: rgba(0, 0, 0, 0);
+  font-family: inherit;
+  font-weight: 400;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
```

Comment:
These styles do not seem to be applying to the demo you provided, please make sure the published version is the most recent.

From DEMO:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/52002856/e18d7afa-ebcb-40bd-b4eb-f06795e2bb1c)